Exception();
            }
            catch(Exception e)
            {
                //System.out.println("inside exception");
                System.out.println(sc.next()+" can't be fitted anywhere.");
            }
         
         
         
         
         
           
       } 
     

        }
    }
}



